src/sso/sso.ts

```diff
@@ -45,4 +46,10 @@ export class SSO {
 
     return data.profile as Profile;
   }
+
+  async promoteDraftConnection({ token }: PromoteDraftConnectionOptions) {
+    await this.workos.post('/draft_connections/convert', {
```

Comment:
We're changing this to `/draft_connections/42/activate` on the other side.

Reply:
is 42 the token?